runners/flink/pom.xml

```diff
@@ -251,6 +251,17 @@
       </exclusions>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-runners-java-fn-execution</artifactId>
```

Comment:
Please add the same dependency to `build.gradle`. It should be simple enough to follow the example of other dependencies. You won't need to do anything special to exclude slf4j because the gradle build handles common dependencies in a cleaner way.